src/Components/Search/SearchBarModal.jsx

```diff
@@ -0,0 +1,47 @@
+import './SearchBarModal.scss';
+
+import React, { Component } from 'react';
+
+class SearchBar extends Component {
+  state = {
+    isModalOpen: false,
+  };
+
+  modalToggle = () => {
+    this.setState(prevState => ({ isModalOpen: !prevState.isModalOpen }));
+  };
```

Comment:
```suggestion
  handleModal = () => {
    this.setState({ isModalOpen: !this.state.isModalOpen });
  };
```
- 이렇게 해도 되지 않을까요? prevState로 해주신 이유가 있으신가요?